src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -531,6 +539,17 @@ private CompareExchangeTombstonesCleanupState GetMaxCompareExchangeTombstonesEta
             return CompareExchangeTombstonesCleanupState.HasMoreTombstones;
         }
 
+        private async Task<bool> RemoveExpiredCompareExchange(TransactionOperationContext context, long nowTicks)
+        {
+            const int batchSize = 1024;
+            if (CompareExchangeExpirationStorage.HasExpired(context, nowTicks) == false)
+                return false;
+
+            var result = await _server.SendToLeaderAsync(new DeleteExpiredCompareExchangeCommand(nowTicks, batchSize, RaftIdGenerator.DontCareId));
```

Comment:
If this command will be executed twice, would you end up with the same result as a single execution?
if the answer is no, you need to provide some `guid` to the command.

Reply:
We need a guid, yes